src/instructlab/lab.py

```diff
@@ -972,33 +955,52 @@ def convert(self, value, param, ctx) -> "torch.device":
     ),
 )
 @click.option(
-    "--model-name",
-    default="instructlab/merlinite-7b-lab",
+    "--dtype",
+    type=click.Choice(["bf16", "fp16", "fp32", "auto"]),
     show_default=True,
-    help="model name to use in training",
+    default="auto",
+    help=("Full half precision training. BF16 is more precise than FP16."),
+)
+@click.option(
+    "--backend",
+    type=click.Choice(["pytorch", "mlx"]),
+    show_default=True,
+    default="pytorch",
+    help=("Backend to use for training. Options are 'mlx' or 'pytorch'."),
+)
+@click.option(
+    "--train-style",
```

Comment:
'--eval' is the most common used parameter for this purpose, set default value to True.